firefox_ui_tests/functional/toolbar/test_access_location_bar.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import By
+
+from firefox_puppeteer.api.keys import Keys
+from firefox_puppeteer.api.places import Places
+from firefox_puppeteer.ui.windows import BrowserWindow
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestAccessLocationBar(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.location_bar = self.browser.navbar.locationbar
+        self.url_bar = self.location_bar.urlbar
+
+        # Purge history
+        self.places.remove_all_history()
+
+        # Navigate to several urls in order to populate history
+        self.test_urls = [
+            'layout/mozilla_projects.html',
+            'layout/mozilla_mission.html',
+            'layout/mozilla.html']
```

Comment:
I do not understand this comment. What difference does the order make in this test?